src/dswx_sar/initial_threshold.py

```diff
@@ -1612,6 +1642,35 @@ def compute_threshold_max_bound(intensity_array,
                       int_sub_mean + int_sub_std * 2]), int_sub_mean, int_sub_std
 
 
+def create_geotiff_with_one_value(outpath, shape, filled_value):
+    """
+    Create a new GeoTIFF filled with zeros for the specified tile.
+    Parameters:
+    ----------
+    tile: tuple
```

Comment:
- Please update the docstring.
- If my understanding is correct, this function should generate a GEOTIFF whose image array is filled with `filled_value`. But the code below multiples `filled_value` with zero-filled array. In that case, the saved GEOTIFF will always be filled with `0`.

Reply:
Thank you for the comments. Good catch. :)